Add minimal template and update build config

viaszkadi opened the pull request.

viaszkadi committed:
Add minimal template and update build config

tib reviewed (approved):
We should revert the LICENSE file, but apart from that it looks good.

viaszkadi committed:
Update copyright holder in LICENSE

Changed the copyright holder from Toucan to Binary Birds Kft. in the MIT license file.
